cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -80,7 +84,7 @@ def __init__(
         self.instance_roles = {}
         self.instance_profiles = {}
         self.compute_security_groups = {}
-        self.shared_storage_ids = {storage_type: [] for storage_type in SharedStorageType}
+        self.shared_storage_mappings = {storage_type: [] for storage_type in SharedStorageType}
```

Comment:
Instead we can keep shared_storage_mappings as a name for the dictionary containing the list of StorageInfo